src/ItemsStorage.php

```diff
@@ -4,16 +4,18 @@
 
 namespace Yiisoft\Rbac\Cycle;
 
+use Cycle\Database\Database;
 use Cycle\Database\DatabaseInterface;
 use Cycle\Database\Injection\Expression;
 use Cycle\Database\Injection\Fragment;
-use Cycle\Database\Table;
 use Yiisoft\Rbac\Item;
 use Yiisoft\Rbac\ItemsStorageInterface;
 use Yiisoft\Rbac\Permission;
 use Yiisoft\Rbac\Role;
 
 /**
+ * **Warning:** Do not use directly! Use with `Manager` from {@link https://github.com/yiisoft/rbac} package.
```

Comment:
How about marking it with `@internal`?

Reply:
@internal was added initially, but we discussed it with @vjik and decided to remove PHPDoc tag only keeping the note.

> `@internal` means that a class can be used only within the package.